CodingPy/Ch13/코딩도장13-7심사문제.py

```diff
@@ -0,0 +1,22 @@
+# 표준 입력으로 가격(정수)과 쿠폰 이름이 각 줄에 입력됩니다. 
+# Cash3000 쿠폰은 3,000원, Cash5000 쿠폰은 5,000원을 할인합니다. 
+# 쿠폰에 따라 할인된 가격을 출력하는 프로그램을 만드세요(input에서 안내 문자열은 출력하지 않아야 합니다).
+# ex)
+# >>> 27000
+# >>> Cash3000
+# 24000
+# >>> 72000
+# >>> Cash5000
+# 67000
+# 문제 제출 코딩도장 Python
+# 답안 제출 2022/01/11 15:16 경준현
+
+Pay = int(input())
+Qpon = input()
```

Comment:
변수의 첫 글자는 소문자를 사용합니다! 또한 qpon이 아니라 coupon으로 하는 게 더 맞는 것 같아요!
```suggestion
pay = int(input())
coupon = input()
```